java_tools/configuration_definition/RusefiConfigGrammar.g4

```diff
@@ -94,7 +94,7 @@ fieldOptionsList
 
 arrayLengthSpec: numexpr (ArrayDimensionSeparator numexpr)?;
 
-autoscale: Autoscale '<' numexpr '>';
+autoscale: Autoscale;
```

Comment:
we probably don't need a separate rule here, we can just put `Autoscale?` down below instead of `autoscale?`